scripts/init-testnet.ts

```diff
@@ -265,36 +265,23 @@ function constructAnnuitySetup(api: ApiPromise) {
 }
 
 async function constructAmmSetup(api: ApiPromise) {
-    // workaround for broken is_exists check - TODO: remove once fixed.
-    // see https://github.com/interlay/interbtc/blob/1a1afa90228f37c9ade4acbda8275c2f5cfe85ce/parachain/runtime/testnet-kintsugi/src/zenlink.rs#L43
-    const isExistsWorkaround = [
-        { Token: "KBTC" }, // foreign assets also need issuance, but faucet already has those
-    ].map((token) => {
-        return api.tx.tokens.setBalance(
-            "a3ckVDnZwjdBhkq1KJodZj3iCgYMseWfLA5fpqExuEpMy8Y5q", // root
-            token,
-            0,
-            1
-        )
-    });
-
-    const prices:Map<string, number> = new Map();
+    const prices: Map<string, number> = new Map();
     prices.set(JSON.stringify({ Token: "KBTC" }), 22842.91);
     prices.set(JSON.stringify({ Token: "KSM" }), 36.05);
     prices.set(JSON.stringify({ Token: "KINT" }), 0.982574);
     prices.set(JSON.stringify({ ForeignAsset: 1 }), 1); // usdt
     prices.set(JSON.stringify({ ForeignAsset: 2 }), 8.94); // movr
-    const decimals:Map<string, number> = new Map();
+    const decimals: Map<string, number> = new Map();
     decimals.set(JSON.stringify({ Token: "KBTC" }), 8);
     decimals.set(JSON.stringify({ Token: "KSM" }), 12);
     decimals.set(JSON.stringify({ Token: "KINT" }), 12);
     decimals.set(JSON.stringify({ ForeignAsset: 1 }), 6); // usdt
     decimals.set(JSON.stringify({ ForeignAsset: 2 }), 18); // movr
-    
+
     const pools: [{ Token: any; }, { Token: any; } | { ForeignAsset: any; }, number, number][] = [
         [
-            { Token: "KBTC" },
             { Token: "KSM" },
+            { Token: "KBTC" },
```

Comment:
Are you sure this is the only one that need to change? Also, this not only impacts `updateRewardSchedule`, but also `createPair` and `addLiquidity`. Won't that create a problem?

Reply:
Yes the index of the `Token` variant will come before `ForeignAsset`